packages/iobio-react-components/src/index.ts

```diff
@@ -31,7 +29,14 @@ export {
 	BamKeys,
 	histogramKeys,
 	percentKeys,
+	type BamContext,
 	type BamHistogramKey,
 	type BamKey,
 	type BamPercentKey,
 } from './constants';
+
+const iobio = async () => {
+	await import('iobio-charts');
+};
```

Comment:
Great workaround! I'd truly like to figure out whether the IIFE example [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#import_a_module_for_its_side_effects_only) would work for us

Reply:
IIFE pattern's been implemented so consumer app can simply do a standard import and the package handles the rest!